healthScore/views.py

```diff
@@ -945,3 +950,66 @@ def update_request_status(request):
         )
 
     return redirect("manage_request")
+
+
+@csrf_exempt
+@require_http_methods(["POST"])
+def send_approval_emails(request):
+    data = json.loads(request.body)
+    emails = data.get("emails", [])
+    requestIds = data.get("requestIds", [])
+
+    hr_ids = HealthRecord.objects.filter(userID=request.user).values_list(
+        "id", flat=True
+    )
+    pdf = view_report(request, hr_ids)
+
+    for email in emails:
+        email_msg = EmailMessage(
+            "Health Record Access Approval Confirmation",
+            "Your request for "
+            + request.user.email
+            + " has been approved. Please find attached health record detail.",
+            EMAIL_HOST_USER,
+            [email],
+        )
+
+        email_msg.attach(
+            "Health_Records_" + request.user.email + ".pdf",
+            pdf.getvalue(),
+            "application/pdf",
+        )
+
+        email_msg.send()
+
+    # set Approved status to selected request ids
+    for id in requestIds:
+        hhar = HealthHistoryAccessRequest.objects.get(id=id)
+        hhar.status = "approved"
+        hhar.save()
+
+    return JsonResponse({"message": "Emails have been sent!"})
+
```

Comment:
Could also please add the test cases for these functions?

Reply:
added!